openedx/features/course_experience/tests/views/test_course_outline.py

```diff
@@ -202,7 +202,7 @@ def test_num_graded_problems(self):
         ([CourseMode.AUDIT, CourseMode.VERIFIED], CourseMode.VERIFIED, False, True),
         ([CourseMode.AUDIT, CourseMode.VERIFIED, CourseMode.MASTERS], CourseMode.MASTERS, False, True),
         ([CourseMode.PROFESSIONAL], CourseMode.PROFESSIONAL, False, True),
-        ([CourseMode.AUDIT, CourseMode.VERIFIED], CourseMode.VERIFIED, True, False),
+        ([CourseMode.AUDIT, CourseMode.VERIFIED], CourseMode.VERIFIED, True, True),
```

Comment:
This makes every argument `True` for `should_display` -- could maybe drop it from the ddt lines.

Reply:
Ah yeah. I was going to say it would be good to have a test with staff = True, but I can just use a different test and remove this one. Will update.